.circleci/config.yml

```diff
@@ -37,6 +37,9 @@ commands:
   check-changed-files-or-halt:
     steps:
       - run: ./scripts/check-file-changes.sh
+  rebase-on-master:
+    steps:
+      - run: git fetch && git rebase origin/master
```

Comment:
Typically a user's origin will be their own fork, not upstream. We will also want to ensure that the master is the absolute latest. To ensure both of these things, I find it is safe to add a new remote and rebase on that. Something like:

```
git remote add upstream https://github.com/influxdata/telegraf
git fetch upstream
git rebase upstream/master
```

Reply:
Good catch, thanks for the suggestion I didn't consider forks. I've updated the PR with the suggested changes!